### PR TITLE
Delete "condition" and "location" for virtual products

### LINKED ISSUE #9623: [BOOM-5752] Delete "condition" and "location" for virtual products
> This issue has been migrated from the Forge. Read the [original ticket here](http://forge.prestashop.com/browse/BOOM-5752).

- _**Reporter:**_ khouloud.belguith
- _**Created at:**_ Thu, 14 Jun 2018 16:31:29 +0200

<p>Hi,</p>

<p>In a virtual product, it is impossible to change the condition of the product to "Use".<br/>
It will be always displayed "New".<br/>
 <span class="image-wrap" style=""><a id="68692_thumb" href="http://forge.prestashop.com/secure/attachment/68692/68692_image-2018-06-14-16-30-24-127.png" title="image-2018-06-14-16-30-24-127.png" file-preview-type="image" file-preview-id="68692" file-preview-title="image-2018-06-14-16-30-24-127.png"></a></span> </p>

<p>Best regards, Khouloud</p>
        
* How to reproduce the issue ?

- Create a virtual product
<br/>
- in the option tab =&gt; Condition &amp; References =&gt; select &quot;Used&quot;
<br/>
- Save
<br/>
- Reload the page =&gt; changes is not saved

### PR DESCRIPTION
**Pull request by kpodemski**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | This improves a new product page CX
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #9623
| Related PRs       | n/a
| How to test?      | Follow instructions from the Issue
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.

### CODE REVIEW

**Review by matks — APPROVED** [review]

**Review by FabienPapet — APPROVED** [review]

**Review by florine2623 — COMMENTED** [review]
Hello @kpodemski ,

Tested the following on _develop_:

**On Product Page V1:**
- ❓No change, it works the same way as before. I supposed this is normal behavior? Could you confirm ?

**On Product page V2:**
- ❓check product created in V1, no change (location and condition are still here)
- creation of virtual product (check BO and FO)
- order creation from FO and BO
- creation of pack with a virtual products (virtual products created from product page V1 and V2)
- modification from virtual to combination/standard
- ❓creation of a standard product and later modification to virtual product. In BO product form OK, but elsewhere in BO and in FO, the 2 fields are still here. Is it normal behavior?
- order creation from FO and BO for all these products

Could you answer the 3 questions?
Other than that, the creation of virtual product with product page V2 looks great ^^

Thanks!

File: src/PrestaShopBundle/Form/Admin/Sell/Product/Specification/SpecificationsType.php [hunk 1]
```diff
@@ -100,7 +106,10 @@ public function configureOptions(OptionsResolver $resolver)
             ->setDefaults([
                 'required' => false,
                 'label' => $this->trans('Details', 'Admin.Catalog.Feature'),
+            ])->setRequired([
```
Comment (leemyongpakvn) [inline src/PrestaShopBundle/Form/Admin/Sell/Product/Specification/SpecificationsType.php:109]: ])
            ->setRequired([
                'product_type',
            ])
for consistency

**Comment by matks** [comment]
Friendly reminder @kpodemski

**Comment by FabienPapet** [comment]
@kpodemski Another friendly reminder 😄

**Comment by matks** [comment]
Hi @kpodemski I believe this PR is now too stale 😉 . I close it for now, just to help navigating old PRs. But you can reopen it anytime you want 😉 obviously

**Pull request closed without merging**